### ISSUE TITLE
Pact Binary Error: Pact unable to identify ruby dependency

### ISSUE THREAD

**Issue by juliusreademyob** [pact-foundation/pact-node#235]
## Software versions

-   **OS**: `Mac OSX 10.14.6`
-   **Library version**: `^10.9.7`
-   **Node Version**: `v12.16.1`

## Expected behaviour
 
Library should identify `/app/node_modules/@pact-foundation/pact-node/standalone/linux-x64-1.85.0/pact/lib/ruby/bin.real/ruby` file. 

## Actual behaviour

`/app/node_modules/@pact-foundation/pact-node/standalone/linux-x64-1.85.0/pact/lib/ruby/bin.real/ruby` file unable to be found.

## Steps to reproduce

So I'm essentially running the contract tests inside a custom docker image. When it goes to run the contract tests and setup the mock server, it will throw the below error about `ruby` not existing. 

However, when I do `ls /app/node_modules/@pact-foundation/pact-node/standalone/linux-x64-1.85.0/pact/lib/ruby/bin.real` within the docker container the `ruby` file is in fact there. So I'm not sure why it's throwing that error. 

Basically the mock server is then never setup and all the tests fail to connect to it. 

## Relevant log files

`[2020-08-06T01:27:05Z] [2020-08-06T01:27:04.923Z] ERROR: pact-node@10.9.7/19 on ab592d5a4bab:
  | [2020-08-06T01:27:05Z]     Pact Binary Error: /app/node_modules/@pact-foundation/pact-node/standalone/linux-x64-1.85.0/pact/lib/ruby/bin/ruby: line 6: /app/node_modules/@pact-foundation/pact-node/standalone/linux-x64-1.85.0/pact/lib/ruby/bin.real/ruby: No such file or directory
`

**Comment by mefellows** [comment]
Have you seen https://github.com/pact-foundation/pact-js#alpine--docker ?

**Comment by juliusreademyob** [comment]
Excellent, that solved everything. Thank you!

**Comment by mefellows** [comment]
Excellent! No worries :)
